### PR TITLE
Add login validation route back in NewDot

### PR DESCRIPTION
**Pull request by roryabraham**
cc @NikkiWines 

### Details
Adds a login validation route that was removed in [this PR](https://github.com/Expensify/App/pull/5736), but should not have been.

### Fixed Issues
$ https://github.com/Expensify/App/issues/6136

### Tests
1. Log in to a validated account without any secondary logins in NewDot
1. Navigate to Settings > Profile and add a secondary login for an account that you have access to
1. Confirm the sent link routes you to NewDot not OldDot
1. Click on the link and confirm your secondary login is validated
1. Repeat steps 1 and 2
1. Click on Resend to resend the validation link to yourself
1. Repeat steps 3 and 4

### QA Steps
1. Complete *Test* steps while logged into an account on NewDot.
    - For step 5 ensure you are logged into the account when you click on the link
1. Complete *Test* steps while logged into an account on NewDot.
    - For step 5 ensure you are logged out of the account when you click on the link

### Tested On

- [ ] Web
- [ ] Mobile Web
- [ ] Desktop
- [x] iOS
- [ ] Android

### CODE REVIEW

**Comment by roryabraham** [comment]
Tested this on iOS on a physical device and it worked 🎉

File: src/pages/ValidateLoginPage.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+};
+
+const defaultProps = {
+    route: validateLinkDefaultProps,
+};
+class ValidateLoginPage extends Component {
+    componentDidMount() {
+        const accountID = lodashGet(this.props.route.params, 'accountID', '');
+        const validateCode = lodashGet(this.props.route.params, 'validateCode', '');
+
```
Comment (NikkiWines) [inline src/pages/ValidateLoginPage.js:22]: NAB: extra line

**Review by NikkiWines — DISMISSED** [review]
👍  tested and works well

**Review by deetergp — DISMISSED** [review]
Code looks good, but you've got a merge conflict 😬

**Comment by roryabraham** [comment]
Updated!

**Review by NikkiWines — APPROVED** [review]
:shipit:

**Pull request merged**